__tests__/src/sagas/accounts/index.js

```diff
@@ -21,6 +22,7 @@ import {
   CHECK_PASSWORD,
   CHECK_PIN_CODE,
   LOGIN,
+  VALID_MNEMONIC_WITH_ACCOUNT,
```

Comment:
I guess it's better to update this naming to `VALIDATE` as well

Reply:
done